File: C - language/Q20.c
#include <stdio.h>

int main() {
    int arr[]={80,30,70,50,20};
    int len=sizeof(arr)/sizeof(arr[0]);
    int max=arr[0];
    int min=arr[0];
    int i,j;
    for (i=0;i<len;i++){
         if(arr[i]>max){
            max=arr[i];
        }
        if(arr[i]<min){
            min=arr[i];
        }
    }
    printf("%d\n",max-min);

    return 0;
}

File: javascript/Q20.js
var arr = [80, 30, 70, 50, 20];
var max=arr[0];
var min=arr[0];

for (i=0;i<arr.length;i++){
    if (arr[i]>max){
       max=arr[i]
    }
    if (arr[i]<min){
        min=arr[i]
    }
}

console.log(max-min)
